src/analysis/processing/qgsalgorithmnearestneighbouranalysis.cpp

```diff
@@ -48,6 +48,7 @@ QString QgsNearestNeighbourAnalysisAlgorithm::groupId() const
 QString QgsNearestNeighbourAnalysisAlgorithm::shortHelpString() const
 {
   return QObject::tr( "This algorithm performs nearest neighbor analysis for a point layer.\n\n"
+                      "The output allows to understand how you data are distributed (clustered, randomly or distributed).\n\n"
```

Comment:
```suggestion
                      "The output allows to understand how your data are distributed (clustered, randomly or distributed).\n\n"
```

Reply:
copy/paste before your suggestion in the other PR :)